dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/PLL/PLL.mo

```diff
@@ -0,0 +1,116 @@
+within Dynawo.Electrical.Controls.PLL;
+
+model PLL "Phase-Locked Loop"
+
+  import Modelica.Blocks;
+  import Modelica.ComplexBlocks;
+  import Modelica.Constants;
+  import Modelica.SIunits;  
+    
+  import Dynawo.Connectors;
+
+  parameter SIunits.PerUnit Kp "PLL voltage calculation proportional gain";
+  parameter SIunits.PerUnit Ki "PLL voltage calculation integrator";
+  parameter SIunits.PerUnit fmin "Lower frequency limit (only positive values!)";
+  parameter SIunits.PerUnit fmax "Upper frequency limit";
+  parameter SIunits.PerUnit fref "Frequency refence value";
+
+  // Inputs:
+  ComplexBlocks.Interfaces.ComplexInput uPu(re(start = u0Pu.re), im(start = u0Pu.im)) "Complex voltage at PCC (pu base UNom)" annotation(
+    Placement(visible = true, transformation(origin = {-116, 29}, extent = {{-17, -17}, {17, 17}}, rotation = 0), iconTransformation(origin = {-101, 51}, extent = {{-9, -9}, {9, 9}}, rotation = 0)));
+  Blocks.Interfaces.RealInput OmegaRefPu(start = Omega0Pu) "Reference frequency of the system. Either connected to the reference machine or the center of inertia frequency or set be constant 1." annotation(
+    Placement(visible = true, transformation(origin = {-117, -70}, extent = {{-18, -18}, {18, 18}}, rotation = 0), iconTransformation(origin = {-101, -47}, extent = {{-9, -9}, {9, 9}}, rotation = 0)));
+  
+  // Outputs:
+  Blocks.Interfaces.RealOutput fmeas "Measured Frequency, PLL (pu)" annotation(
```

Comment:
to be consistant with the naming convention call it omegaPLLPu?

Reply:
oki, done